cluster/oke.go

```diff
@@ -536,5 +552,56 @@ func (o *OKECluster) ListNodeNames() (nodeNames pkgCommon.NodeNames, err error)
 
 // RbacEnabled returns true if rbac enabled on the cluster
 func (o *OKECluster) RbacEnabled() bool {
-	return o.modelCluster.RbacEnabled
+	return true
+}
+
+// setClusterAdminRights creates a cluster role binding which gives admin
+// rights to the user ocid specified in the secret used to create the cluster
+func (o *OKECluster) setClusterAdminRights(name string) (err error) {
```

Comment:
No need to define a name `err` in advance as you don't use it anywhere (the first `err :=` redeclares the variable and you return a value everywhere)